modules/ROOT/pages/changelog.adoc

```diff
@@ -17,13 +17,13 @@ NOTE: This is the {productname} Community version changelog. For information abo
 === Added
 * New `tree` component that can be used in dialog body panel.
 * `renderUI` property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
-* New `isEditable` API to `editor.selection` that returns true or false if the current selection is editable.
-* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
-* New `setText` and `setIcon` methods added to menu button and toolbar button API.
+* New `editor.selection.isEditable` API method that returns whether the current selection is editable.
+* New `editor.dom.isEditable` API method that returns whether the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button APIs.
 * New `highlight_on_focus` option which enables highlighting the content area on focus.
 * New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.
-* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
-* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* Added `skip_focus` check to the ToggleToolbarDrawer command.
```

Comment:
No idea what `skip_focus` is

Reply:
its an optional argument for certain commands which involve UI elements, when false or null the editor will be focused after the command is run, but when set to true this focus step will be skipped.